### PR TITLE
fix(eslint rule and subsequent changes to the codebase): eslint rule no-console added

### LINKED ISSUE #2307: Disallow console log in ESLint
## 🐛 Bug Report

As a followup to https://github.com/HospitalRun/hospitalrun-frontend/issues/2286 I thought it might be desirable to not allow any log messages in the application using an ESlint rule https://eslint.org/docs/rules/no-console

## Proposed changes 
* Remove any `console` call from the application
* Add a rule in the ESLint config that would make the linter fail if there is a `console` call in the application

Is there any good reason why `console` logs should be kept in the application?

### PR DESCRIPTION
**Pull request by tobireuen**
Fixes #2307 

Changes proposed in this pull request:

- add eslint rule 'no-console' to throw errors whenever calls to the console object are made
- remove all calls to the console object OR allow them on file level for specific cases

Newly added dependencies with Bundlephobia links:

NONE

### CODE REVIEW

File: couchdb-cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/zsh
```
Comment (fox1t) [inline couchdb-cleanup.sh:1]:
```suggestion
#!/bin/sh
```

Comment (fox1t) [reply couchdb-cleanup.sh:1]: Not everyone has zsh installed

File: couchdb-init.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/zsh
```
Comment (fox1t) [inline couchdb-init.sh:1]:
```suggestion
#!/bin/sh
```

**Review by fox1t — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: couchdb-cleanup.sh (hunk 1 above)
Comment (tobireuen) [reply couchdb-cleanup.sh:1]: Oops, I didn't even mean to commit this and couchdb-init.sh. These were just helper scripts for my environment.

Is there a way to remove them from the pull request?

Comment (fox1t) [reply couchdb-cleanup.sh:1]: Actually I like them :)

Maybe you can fix that and move them in a dedicated folder?

Comment (tobireuen) [reply couchdb-cleanup.sh]: They are quite handy but shouldn't be part of this pull request. I'll go ahead and create another one for that ;)

Comment (fox1t) [reply couchdb-cleanup.sh]: Please do it! :)

**Review by fox1t — APPROVED** [review]
LGTM

**Pull request merged**